### ISSUE TITLE
Improve the Hero UI for small viewport

### ISSUE THREAD

**Issue by rahuldahal** [askbuddie/askbuddie-bot-docs#19]
While on viewport, smaller than 768px, the hero's design element(red circle) sits behind the call-to-action button.
Edit [this](https://github.com/askbuddie/askbuddie-bot-docs/blob/1846d6d54b1628e1585050a95f8a44ce6b2e33c1/styles/custom.css#L56) CSS block to include `@media (min-width: 768px) { ... }` and make sure the circle appears at 
![image](https://user-images.githubusercontent.com/44515005/131664948-7b0d2e4b-9d36-4135-a325-798755b78199.png) when the screen is smaller than 768px. 

But, when the screen is wider, make sure it appears at 
![image](https://user-images.githubusercontent.com/44515005/131665311-a548d10e-29e3-4cd8-937a-e8c27689025e.png)

**Additional context**
Basically, the design should go from this,
![image](https://user-images.githubusercontent.com/44515005/131665452-f1e07511-8a56-4ced-bb2a-5271b5412a78.png)

to this,
![image](https://user-images.githubusercontent.com/44515005/131665536-24fc333d-7dfc-43fd-8b2c-fb51394e7bcc.png)
